packages/components/pie-chip/src/chip.scss

```diff
@@ -109,18 +79,18 @@
         }
     }
 
-    &.c-chip--disabled {
-        --chip-bg-color: var(--dt-color-disabled-01);
+    &.disabled {
+        --bg-color: var(--dt-color-disabled-01);
         --chip-color: var(--dt-color-content-disabled);
         --chip-cursor: not-allowed;
         --chip-close-btn-cursor: not-allowed;
 
         &.c-chip--ghost {
-            --chip-bg-color: transparent;
+            --bg-color: transparent;
         }
     }
 
-    &.c-chip--dismissible {
+    &.dismissible {
```

Comment:
I think I understand updating the disabled classes, but please could you explain why the dismissible one should change too?

Reply:
Never mind, I got to the mixin 😄